### PR TITLE
Add ceph config file for kolla

### PR DESCRIPTION
**Pull request by ktibi**

### CODE REVIEW

File: ansible/roles/kolla-openstack/vars/main.yml [hunk 1]
```diff
@@ -77,3 +77,8 @@ kolla_openstack_custom_config:
     dest: "{{ kolla_node_custom_config_path }}/swift"
     patterns: "*"
     enabled: "{{ kolla_enable_swift }}"
+  # Ceph.
+  - src: "{{ kolla_extra_config_path }}/ceph"
+    dest: "{{ kolla_node_custom_config_path }}/ceph"
+    patterns: "*"
+    enabled: "{{ kolla_enable_ceph }}"
```
Comment (markgoddard) [inline ansible/roles/kolla-openstack/vars/main.yml]: Minor nit: preserve alphabetical order.

Comment (markgoddard) [reply ansible/roles/kolla-openstack/vars/main.yml]: I don't think we have kolla_enable_ceph yet, unless you have added it?

**Comment by markgoddard** [comment]
Looks good, but depends on adding kolla_enable_ceph.

**Comment by ktibi** [comment]
kolla_enable_ceph is already defined in etc/kayobe/kolla.yml no ?

**Comment by markgoddard** [comment]
Defaults are set here: https://github.com/stackhpc/kayobe/blob/master/ansible/group_vars/all/kolla#L239

**Review by markgoddard — APPROVED** [review]
Great, thanks.

**Pull request merged**